devfile.yaml

```diff
@@ -0,0 +1,24 @@
+schemaVersion: 2.1.0
+metadata:
```

Comment:
I think it's missing the attribute to deploy extensions in the user container ?

Reply:
The [sidecar image](https://github.com/eclipse-che/che-plugin-registry/blob/main/sidecars/python/Dockerfile) is a bit different then user container image. So what is the right way: deploy extension to the sidecar image or modify user container image?

Reply:
^^ @benoitf

Reply:
I think for DevWorkspaces, default is to have extensions running in the user container (no longer need to bring the sidecar image)

So it should work with the universal devfile base image

Reply:
but it can be a step 2 (use sidecar image as user image) or step 3 (use universal base image) https://github.com/eclipse/che/issues/20507

but as we're moving to it, it may be a good step to report issues on devfile base image if it doesn't work well

Reply:
The attribute was added. For this devfile we can use the universal base image. 
But for another python samples we probably will need some additional tools/libraries.

Reply:
Will know about that a bit later :-)